mmdet/core/visualization/image.py

```diff
@@ -21,11 +21,23 @@ def color_val_matplotlib(color):
     Returns:
         tuple[float]: A tuple of 3 normalized floats indicating RGB channels.
     """
+    if isinstance(color, list):
+        color = tuple(color)
     color = mmcv.color_val(color)
     color = [color / 255 for color in color[::-1]]
     return tuple(color)
 
 
+def palette_val(palette):
```

Comment:
OK, I will change the style quickly.

Reply:
And it is better to use the palette in panopticapi.
Because in panopticapi, each category corresponds to a color. Unfortunately, there are only palettes for coco and cityscapes.